### PR TITLE
Update comms guidelines

### PR DESCRIPTION
**Pull request by mrbobbytables**
This PR updates all our comms documents with Hugo metadata, applies the style guide and fixes several out of date things.
Beyond that, content is has not been changed.

After this is merged, these docs should be surfaceable on the contributor site

### CODE REVIEW

File: communication/requesting-survey.md [hunk 1]
```diff
@@ -1,79 +1,153 @@
+---
+title: "Community Survey Requests"
+description: |
+  Outlines the process and workflow for requesting surveys on behalf of the
+  Kubernetes project.
+---
```
Comment (mrbobbytables) [inline communication/requesting-survey.md:6]: **NOTE:** Outside of this addition, the rest of the changes to this doc were a no-op. Content was not changed at all, just formatting.

**Review by idvoretskyi — APPROVED** [review]
/lgtm

/hold for others to approve.

**Pull request merged**